app/models/taxon_facet.rb

```diff
@@ -0,0 +1,94 @@
+class TaxonFacet < DropdownSelectFacet
+  TOP_LEVEL_TAXON_KEY = 'top_level_taxon'.freeze
```

Comment:
In the domain of content schemas and links we call this a `level_one_taxon` – how about we adopt that language here too?